bin/conjur-cli.rb

```diff
@@ -59,6 +59,12 @@ def test_select
   c.action do |global_options,options,args|
     account = options[:account]
 
+    # Sets all the `unset` Environment Variables to nil. This ensures that nil
+    # checks are sufficient to verify the usage of an environment variable
+    ENV.each_pair do |(k,v)|
+      ENV[k] = nil if v =~ /^\s*$/ # is all whitespace
+    end
+
```

Comment:
@h-artzi Is this only relevant for OSS?

DAP doesn't use `conjurctl` to run the app, so this wouldn't work there.